### ISSUE TITLE
News page - please use revised thumbnail content

### ISSUE THREAD

**Issue by ashlchap** [reskillamericans/website#45]
Note to Mike: 
Please replace the thumbnail content for each blog (which were from email promotions before the event) with the blog recaps that I wrote. If this is being pulled directly from our youtube channel, then can we replace the youtube content? You can find them all below:

https://netorgft7227627.sharepoint.com/:w:/r/sites/RSA2022/_layouts/15/Doc.aspx?sourcedoc=%7B1CF13C28-C97A-45E9-B89C-84911F51A9E0%7D&file=Blogs%20content%20from%20June%2028%202021%20onward.docx&action=default&mobileredirect=true&cid=1e86f1e0-fcb6-43ee-b0bf-70745b511e33

**Comment by mckoss** [comment]
I'm copying these over as is.  Some comments:

1. I removed text that asked people to click to visit the video (since the text will be ON the video).
2. I was following a standard title format before:
    Series Name #N | Guest Name - Title
    I find it useful to scan for series numbers in playlists, etc.  But I took that out and went with your titles.
3. Titles can have no more than 100 characters on YouTube - so I had to shorten some.
4. YouTube does not support hyper-linked text.  So I generally placed any links at the bottom of the description.
